### DISCUSSION TITLE
[ALL] 사장모드에서 고객 조회의 정렬 조건 관련

### DISCUSSION THREAD

**Discussion by gitchannn** [woowacourse-teams/2023-stamp-crush#843] [General]
[의견 요청합니다]

사장모드에서 고객 조회의 정렬 조건 관련해서 

1. 최근 방문 순
2. 방문 횟수
3. 리워드 수 
4. 스탬프 순 

위의 정렬 조건 중 어떤 것을 도입해야 할 지에 대한 의견을 구합니다.

**Comment by yenawee** [comment]
다 도입하면 안돼여?

제 개인적인 중요도 순위는
1. 최근 방문 순
2. 방문 횟수 순
3. 스탬프 개수 순
4. 리워드 개수 순

**Reply by gitchannn** [reply]
다 도입하려면 제나씨가 3,4번 특공대로 투입되는 것도 고려해 보셨으면 합니다❤️

**Comment by kangyeongmin** [comment]
`최근 방문순`과`방문횟수` 2개만 있으면 될 것 같습니다.

`최근방문순`은 고객님이 요청하셨고, 
단골의 기준은 리워드, 스탬프 순이 아닌 `방문횟수`이기 때문입니다.

**Comment by gitchannn** [comment]
저도 역시 최근 방문순과 방문횟수 2개만 있으면 될 것 같습니다.

왜냐하면 제가 사장님이라면 그것만 궁금할 것 같기 때문입니다.

**Comment by jundonghyuk** [comment]
`1` `2` thanks

**Comment by jundonghyuk** [comment]
영호차 -> `1234` thx

**Reply by gitchannn** [reply]
you're miimiiiiiiiii chemical hadi boy~

**Comment by regularPark** [comment]
# ☝🏻✌️
